src/_DataStructures_/DoublyLinkedList/doublyLinkedList.test.js

```diff
@@ -0,0 +1,40 @@
+const DLL = require('.');
+
+describe('Doubly Linked List', () => {
+  const doublyLinkedList = new DLL();
+
+  it('create DLL', () => {
```

Comment:
'create DLL' -> **'It should create a DLL'**

It's all about Behavior Driven Testing, hence it starts with or contains **should**